eng/Versions.props

```diff
@@ -14,7 +14,7 @@
   </PropertyGroup>
   <PropertyGroup>
     <VersionPrefix>3.1.0</VersionPrefix>
-    <PreReleaseVersionLabel>preview3</PreReleaseVersionLabel>
+    <PreReleaseVersionLabel></PreReleaseVersionLabel>
```

Comment:
You need to add an additional property here 

    <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">true</StabilizePackageVersion>

Reply:
And PreReleaseVersionLabel should be rtm